__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -0,0 +1,1887 @@
+    await GenesisWalletHelper.injectFunds(hWallet1.getAddressAtIndex(0), 10);
+    const { hash: tokenUid } = await createTokenHelper(
+      hWallet1,
+      'Delegate Token 2',
+      'DTK2',
+      100,
+    );
+
+    // Creating another mint authority token on the same wallet
+    const { hash: duplicateMintAuth } = await hWallet1.delegateAuthority(
+      tokenUid,
+      'mint',
+      hWallet1.getAddressAtIndex(1),
+      { createAnother: true }
+    );
+    await waitForTxReceived(hWallet1, duplicateMintAuth);
+
+    // Confirming two authority tokens on wallet1
+    let auth1 = await hWallet1.getMintAuthority(tokenUid, { many:true });
+    expect(auth1).toMatchObject([
+      {
+        txId: duplicateMintAuth,
+        index: 0,
+        address: hWallet1.getAddressAtIndex(1),
+        authorities: 1
+      },
+      {
+        txId: duplicateMintAuth,
+        index: 1,
+        address: expect.any(String),
+        authorities: 1
+      },
+    ]);
+
+    // Now having two mint authority tokens on wallet 1, delegate a single one to wallet 2
+    const { hash: delegateMintAuth } = await hWallet1.delegateAuthority(
+      tokenUid,
+      'mint',
+      hWallet2.getAddressAtIndex(1),
+      { createAnother: false }
+    );
+    await waitForTxReceived(hWallet1, delegateMintAuth);
+
+    // Confirming only one authority token was sent from wallet1 to wallet2
+    auth1 = await hWallet1.getMintAuthority(tokenUid, { many:true });
+    expect(auth1).toMatchObject([
+      {
+        txId: duplicateMintAuth,
+        index: expect.any(Number),
+        address: expect.any(String),
+        authorities: 1
+      },
+    ]);
+
+    // Confirming one authority token was received by wallet2
+    const auth2 = await hWallet2.getMintAuthority(tokenUid, { many:true });
+    expect(auth2).toMatchObject([
+      {
+        txId: duplicateMintAuth,
+        index: expect.any(Number),
+        address: expect.any(String),
+        authorities: 1
+      },
+    ]);
+  });
+
+  it('should delegate melt authority to another wallet while keeping one', async () => {
+    // Creating a Custom Token on wallet 1
+    await GenesisWalletHelper.injectFunds(hWallet1.getAddressAtIndex(0), 10);
+    const { hash: tokenUid } = await createTokenHelper(
+      hWallet1,
+      'Delegate Token 2',
+      'DTK2',
+      100,
+    );
+
+    // Creating another melt authority token on the same wallet
+    const { hash: duplicateMeltAuth } = await hWallet1.delegateAuthority(
+      tokenUid,
+      'melt',
+      hWallet1.getAddressAtIndex(1),
+      { createAnother: true }
+    );
+    await waitForTxReceived(hWallet1, duplicateMeltAuth);
+
+    // Confirming two authority tokens on wallet1
+    let auth1 = await hWallet1.getMeltAuthority(tokenUid, { many:true });
+    expect(auth1).toMatchObject([
+      {
+        txId: duplicateMeltAuth,
+        index: 0,
+        address: hWallet1.getAddressAtIndex(1),
+        authorities: 2, // XXX: Should be 1, but this returns 2
```

Comment:
Can you open an issue to investigate why this happens?

Reply:
We found out this was because the `authorities` field actually returns the `value` of the authority output, not the _amount of authorities_ it has. In this case, a melt byte value is `2` and it's correct.